packages/core/include/combo/data/items.h

```diff
@@ -306,6 +306,9 @@
 #define ITEM_MM_BOOTS_HOVER                 0xb2
 #define ITEM_MM_TUNIC_GORON                 0xb3
 #define ITEM_MM_TUNIC_ZORA                  0xb4
+#define ITEM_MM_HAMMER                      0xb5
+#define ITEM_MM_CUSTOM_MAX                  0xb6
+
 /* If we want to use values above 0xb5 we need to fix a lot of code related to magic arrows */
```

Comment:
Can this comment be updated to reflect the new safest upper value to use?

Reply:
I will update it. For reference, the safest upper value is now ITEM_MM_CUSTOM_MAX

Reply:
Oh I thought that the changes in kaleido_scope.S and z_parameter.S made it automatically adjust the magic arrows issue if you increase this ITEM_MM_CUSTOM_MAX value?

Reply:
That is correct, if you increase it the safest upper value also increases. I updated the comment to reflect that

Reply:
Okay that's what I thought. What I meant to ask for was what's the new safest upper limit for ITEM_MM_CUSTOM_MAX? Magic arrows use 3 slots i guess, but I assume 0xFF is not usable. Can magic arrows use 0xFE, 0xFD and 0xFC which makes 0xFB the highest safest value?

Reply:
Oh sorry. Items greater than or equal to 0xF0 are reserved for special purposes and button actions, and kaleidoscope will attempt to avoid them. For safety I'd suggest going no higher than 0xE0, though the upper limit may be 0xED. You would have to add quite a few items to get to a point where this would be an actual problem, but I guess it is possible.